.github/workflows/project_issues.yml

```
@@ -1,18 +1,83 @@
-name: Project Automation - move issues
+name: Project Automation - Move Issues
 
 on:
+  pull_request:
+    types: [opened, synchronize, reopened]
   issues:
     types: [labeled]
 
+env:
+  PAT_TOKEN: ${{ secrets.PAT_TOKEN }}
+  PROJECT_URL: https://github.com/orgs/Royal-Navy/projects/9
+
 jobs:
-  Move_labelled_issues:
+  Test_issue_mover:
     runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
     steps:
-      - name: Move small/med labeled issues to Candidates column
-        uses: Royal-Navy/design-system-moveissue-action@master
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Create test issue
+        id: create_issue
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.PAT_TOKEN }}
+          script: |
+            const issue = await github.rest.issues.create({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              title: 'Test issue for PR #' + context.issue.number,
+              body: 'This is a test issue created to verify the issue mover script.',
+              labels: ['Size: Small']
+            });
+            console.log('Test issue created:', issue.data.html_url);
+            core.setOutput('issue_number', issue.data.number);
+            core.setOutput('issue_node_id', issue.data.node_id);
+
+      - name: Run issue mover script
+        uses: actions/github-script@v6
```

Would be good to update to v7 throughout:

```suggestion
        uses: actions/github-script@v7
```